src/libraries/System.IO.FileSystem/src/System/IO/File.cs

```diff
@@ -114,7 +114,7 @@ public static void Delete(string path)
         // given by the specified path exists; otherwise, the result is
         // false.  Note that if path describes a directory,
         // Exists will return true.
-        public static bool Exists(string? path)
+        public static bool Exists([NotNullWhen(true)] string? path)
```

Comment:
You'll need to add a `using System.Diagnostics.CodeAnalysis;` to the top of the file.

Reply:
Done.